sdks/go/cmd/beamctl/cmd/artifact.go

```diff
@@ -42,10 +42,13 @@ var (
 		RunE:  listFn,
 		Args:  cobra.NoArgs,
 	}
+
+	token string
```

Comment:
Would recommend to rename this to stagingToken to make clear that this is not the retrieval token.

Reply:
Done.